### PR TITLE
Add action pod lib lint

### PR DESCRIPTION
**Pull request by thierryxing**
Add action pod lib lint

### CODE REVIEW

File: fastlane/lib/fastlane/actions/pod_lib_lint.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      def self.output
+      end
+
+      def self.return_value
+        nil
+      end
+
+      def self.authors
+        ["Thierry Xing"]
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/pod_lib_lint.rb]: This should be your Twitter or GitHub username

Comment (thierryxing) [inline fastlane/lib/fastlane/actions/pod_lib_lint.rb]: OK

Comment (thierryxing) [inline fastlane/lib/fastlane/actions/pod_lib_lint.rb]: I have already changed authors to my Github username, thanks

**Comment by marcelofabri** [comment]
This probably should use `bundle exec` as the `cocoapods` action: https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/cocoapods.rb#L17

File: fastlane/spec/actions_list_spec.rb [hunk 2]
```diff
@@ -61,7 +61,7 @@
       count = 0
       Fastlane::ActionsList.all_actions("nothing special") { count += 1 }
       expect(count).to be > 40
-      expect(count).to be < 80
```
Comment (KrauseFx) [inline fastlane/spec/actions_list_spec.rb]: Please remove this change from the PR and rebase master, it should work now 👍

**Comment by KrauseFx** [comment]
Thanks for the PR @thierryxing, could you also include the `bundle exec` prefix, similar to how CocoaPods does it? https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/cocoapods.rb#L17

File: fastlane/spec/actions_list_spec.rb (hunk 2 above)
Comment (thierryxing) [inline fastlane/spec/actions_list_spec.rb]: OK, no problem!

**Comment by thierryxing** [comment]
OK, I'll add it right now.

**Comment by thierryxing** [comment]
All done!

File: fastlane/docs/Actions.md [hunk 3]
```diff
@@ -2487,4 +2487,20 @@ To run [danger](https://github.com/danger/danger) add this to your `Fastfile`
 lane :danger do
   danger
 end
+=======
```
Comment (KrauseFx) [inline fastlane/docs/Actions.md]: What's this about?

Comment (KrauseFx) [inline fastlane/docs/Actions.md]: This breaks markdown

File: fastlane/lib/fastlane/actions/pod_lib_lint.rb [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          sources = params[:sources].join(",")
+          command << "--sources='#{sources}'"
+        end
+
+        if params[:allow_warnings]
+          command << "--allow-warnings"
+        end
+
+        result = Actions.sh(command.join(' '))
+        Helper.log.info "Pod lib lint Successfully ⬆️ ".green
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/pod_lib_lint.rb]: Great choice of emoji

**Comment by KrauseFx** [comment]
Looks great, however the Actions.md change is broken. Please update the PR and let me know once it's ready to merge 👍

**Comment by thierryxing** [comment]
Got it!

**Comment by thierryxing** [comment]
All Done! 
It's ready to merge.
Thank you

File: fastlane/docs/Actions.md (hunk 3 above)
Comment (thierryxing) [inline fastlane/docs/Actions.md]: Sorry!. It maybe happened when handling git rebase conflict, and I just fixed it.

**Comment by KrauseFx** [comment]
This is great, thank you so much! Your action will be in the next release. Thanks 
@thierryxing 👍

**Comment by thierryxing** [comment]
Thank you very much for helping me finish this PR.@KrauseFx,@marcelofabri,@asfalcone
Fastlane is an awesome tool, and it has been helping us to build our own CI and CR system and save quite a lot of time for us. 
It is my great pleasure to contribute back to Fastlane, I will keep doing this as possible as I can.

**Pull request merged**